app/javascript/react/screens/App/Overview/components/Migrations/MigrationsInProgressCard.js

```diff
@@ -52,12 +53,16 @@ const MigrationsInProgressCard = ({ plan, allRequestsWithTasks, reloadCard, hand
   // UX business rule 2: aggregrate the tasks across requests reflecting current status of all tasks,
   // (gather the last status for the vm, gather the last storage for use in UX bussiness rule 3)
   const tasks = {};
-  requestsOfAssociatedPlan.forEach(request => {
-    request.miq_request_tasks.forEach(task => {
-      tasks[task.source_id] = tasks[task.source_id] || {};
-      tasks[task.source_id].completed = task.status === 'Ok' && task.state === 'finished';
-      tasks[task.source_id].virtv2v_disks = task.options.virtv2v_disks;
-    });
+  let tasksOfPlan = {};
+  if (requestsOfAssociatedPlan.length > 0) {
+    tasksOfPlan = getMostRecentVMTasksFromRequests(requestsOfAssociatedPlan, plan.options.config_info.vm_ids);
+  } else if (mostRecentRequest) {
+    tasksOfPlan = mostRecentRequest.miq_request_tasks;
+  }
+  tasksOfPlan.forEach(task => {
+    tasks[task.source_id] = tasks[task.source_id] || {};
+    tasks[task.source_id].completed = task.status === 'Ok' && task.state === 'finished';
```

Comment:
we still haven't extracted these task statuses to constants... since the number of places these are used is increasing, and we have pre/post playbook support upcoming, i think it would be nice to move those to constants in a future PR